scripts/check_unforked.sh

```diff
@@ -14,6 +14,16 @@ check_import() {
   fi
 }
 
+check_blacklist() {
+  local path="$1"
+
+  local result=$(grep -Hne '\("github.com/google/certificate-transparency-go/trillian/ctfe.*"\)' "$path")
+  if [[ ! -z "${result}" ]]; then
+    echo "$result - importing CTFE into generic code is not allowed"
```

Comment:
maybe a smidgeon more specific: "importing CT-specific code into generic Trillian code is not allowed"

Reply:
Done.